Library/Formula/h2.rb

```diff
@@ -17,6 +17,9 @@ def install
     # Remove windows files
     rm_f Dir["bin/*.bat"]
 
+    h2_script = File.read("bin/h2.sh").gsub("\r\n", "\n")
+    File.open("bin/h2.sh", "w") {|f| f.write h2_script}
```

Comment:
Can just do `(bin/"h2.sh").write(File.read("bin/h2.sh").gsub("\r\n", "\n"))` for a one-liner